script/test_in_container/Dockerfile

```diff
@@ -0,0 +1,32 @@
+# This Dockefile creates image where all Kompose tests can be run
+
+FROM registry.centos.org/centos/centos:7
+MAINTAINER Tomas Kral <tkral@redhat.com>
```

Comment:
MAINTAINER has been deprecated in 1.13 of Docker :(.

Better to use `LABEL maintainer Tomas Kral <tkral@redhat.com>` or remove it all-together